docs/manual/installation/quickstart.en.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Install Contao - Quick start"
+description: "The first steps to installing Contao with the Contao Manager."
+aliases:
+- /en/installation/quickstart/
+weight: 90
+hidden: true
+---
+
+We assume here that you want to install either the latest version or the [Long Term Support Version](https://docs.contao.org/manual/en/installation/update-contao/#long-term-support-versions) with the Contao Manager.
+This is the easiest and recommended way for beginners.
+
+## osting configuration
+
+You configure the hosting via the admin panel of your hosting provider.
+
+### Root directory (document root)
+
+In Contao, all publicly accessible files are located in the `/public` subfolder of the installation. Create
+folder `public` and set the root directory (document root) of the installation to this subfolder via the admin panel
+of the hosting provider to this subfolder.
+
+**Example:** `example.com` points to the directory `/www/example/public`.
+
+### Database
+
+Contao requires a [MySQL](../../installation/system-requirements/#mysql-minimum-requirements) database for operation, which you 
+should ideally create right away. The access data will be needed later.
+
+
+## Installing the Contao Manager
+
+The [Contao Manager](../../installation/contao-manager/) consists of a single file which can be downloaded from 
+[contao.org](https://contao.org/en/download). After a successful download you will receive a file `contao-manager.phar`. 
+Transfer this file to the `public` directory on your web server.
+
+{{% notice info %}}
+`.phar` files are not executed by all hosting providers. For best compatibility, add the `.php` file extension 
+<b>after the upload</b> (rename the file <b>on the server</b> to `contao-manager.phar.php`).
+{{% /notice %}}
+
+## Call Contao Manager
+
+Then call up the URL `www.example.com/contao-manager.phar.php` with your browser, replacing `www.example.com` with your domain.
+You should now see the welcome page of the Contao Manager.
+
+![Welcome page of the Contao Manager]({{% asset "images/manual/installation/de/willkommensseite-des-contao-managers.png" %}}?classes=shadow)
+
+### Basic configuration
+
+Before you install Contao, you need to configure the manager itself. To do this, create a new user by
+assigning a username and password. Username and password are independent of the later Contao installation!
```

Comment:
```suggestion
Before you install Contao you need to configure the manager itself. To do this create a new user by
assigning a username and password. Username and password are independent of the later Contao installation!
```

I am not an expert on English punctuation - there is likely way more that needs to be changed ;)

Reply:
Me neither